proto/iotics/api/meta.proto

```diff
@@ -82,29 +80,6 @@ enum SparqlResultType {
   RDF_NTRIPLES = 5;
 }
 
-// SparqlQueryRequest describes a SPARQL query.
-message SparqlQueryRequest {
```

Comment:
Note to reviewers: This is not re-used/modified since `ExplorerQuery` still uses it.